chef/data_bags/crowbar/migrate/network/101_add_bonding_miimon_xmit_hash_policy.rb

```diff
@@ -0,0 +1,19 @@
+def upgrade(ta, td, a, d)
+  unless a["teaming"].key? "miimon"
+    a["teaming"]["miimon"] = ta["teaming"]["miimon"]
+  end
+  unless a["teaming"].key? "xmit_hash_policy"
+    a["teaming"]["xmit_hash_policy"] = ta["teaming"]["xmit_hash_policy"]
+  end
+  return a, d
+end
+
+def downgrade(ta, td, a, d)
+  unless ta["teaming"].key? "miimon"
+    a["teaming"].delete "miimon"
+  end
+  unless ta["teaming"].key? "xmit_hash_policy"
+    a["teaming"].delete "xmit_hash_policy"
+  end
+  return a, d
```

Comment:
I think that's important. I tripped over it when testing downgrade some days ago.